engine-comparison/dispatcher.sh

```diff
@@ -66,6 +66,17 @@ build_benchmark_using() {
   mkdir $SEND_DIR
 
   cp ${BUILDING_DIR}/${BENCHMARK}-${FUZZING_ENGINE} $SEND_DIR
+
+  #TODO: make these gsutil cp?
```

Comment:
I'm pretty sure `rsync` can sync individual files just fine.  Is this a `gsutil rsync` limitation?